app/Migrations/Version20240402125210.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240402125210 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->createTable('movie_mapping');
+
+        $table->addColumn('event_id', Types::GUID)->setLength(36)->setNotnull(true);
+        $table->addColumn('movie_id', Types::STRING)->setNotnull(true);
+
+        $table->setPrimaryKey(['event_id']);
+        $table->addIndex(['movie_id']);
```

Comment:
I think this index is unneeded. SInce the composite unique index covers searching on movie_id

Reply:
I'm not sure about this. I think the combination index is separate from the single-column index.

Reply:
https://planetscale.com/learn/courses/mysql-for-developers/indexes/duplicate-indexes?autoplay=1

Reply:
In short: if the column is the first column inside the unique index no separate index is needed.

See: https://dev.mysql.com/doc/refman/8.0/en/mysql-indexes.html

>If the table has a multiple-column index, any leftmost prefix of the index can be used by the optimizer to look up rows. For example, if you have a three-column index on (col1, col2, col3), you have indexed search capabilities on (col1), (col1, col2), and (col1, col2, col3). For more information, see [Section 10.3.6, “Multiple-Column Indexes”](https://dev.mysql.com/doc/refman/8.0/en/multiple-column-indexes.html).